pkg/gds/admin.go

```diff
@@ -132,6 +132,7 @@ func (s *Admin) setupRoutes() (err error) {
 	// Application Middleware
 	s.router.Use(ginzerolog.Logger("gin"))
 	s.router.Use(gin.Recovery())
+	s.router.Use(s.Available())
```

Comment:
Any particular reason why this was moved? Does the order of these statements matter or is it mainly cosmetic?

Reply:
I think it actually does matter; the order of the middleware application determines the order middleware is applied.